crates/executor/src/blockchain_tree/config.rs

```diff
@@ -0,0 +1,55 @@
+//! Blockchain tree configuration
+
+/// The configuration for the blockchain tree.
+#[derive(Clone, Debug)]
+pub struct BlockchainTreeConfig {
+    /// Finalization windows. Number of blocks that can be reorged
+    max_reorg_depth: u64,
+    /// Number of block after finalized block that we are storing. It should be more then
+    /// finalization window
+    max_blocks_in_chain: u64,
+    /// For EVM's "BLOCKHASH" opcode we require last 256 block hashes. So we need to specify
+    /// at least `additional_canonical_block_hashes`+`max_reorg_depth`, for eth that would be
+    /// 256+64.
+    num_of_additional_canonical_block_hashes: u64,
+}
+
+impl Default for BlockchainTreeConfig {
+    fn default() -> Self {
+        Self {
+            max_reorg_depth: 64,
+            max_blocks_in_chain: 65,
+            num_of_additional_canonical_block_hashes: 256,
```

Comment:
can we add a note about how we came up with these or that they're just assumptions